runtime/tests/interpreter/metatype_test.go

```diff
@@ -657,11 +657,11 @@ func TestInterpretGetType(t *testing.T) {
 		{
 			name: "array",
 			code: `
-              let result = [].getType()
```

Comment:
Maybe leave the existing case as-is and add the new code as a new test case

Reply:
This particular test checks the runtime behavior. But now that empty arrays without contextual type are giving semantic errors, this test-case is no longer valid.
I added a separate test to check the semantic error: https://github.com/onflow/cadence/blob/935fa5c9ba493e8d86c6d1a79ba8798b7e5e82a8/runtime/tests/checker/type_inference_test.go#L915-L924

Reply:
Ah, makes sense! 👍